src/Cache/NullCache.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace DevMob\Postcodes\Cache;
+
+use Psr\SimpleCache\CacheInterface;
+
+class NullCache implements CacheInterface
+{
+    /**
+     * Fetches a value from the cache.
+     *
+     * @param  string $key The unique key of this item in the cache.
+     * @param  mixed $default Default value to return if the key does not exist.
+     * @return mixed The value of the item from the cache, or $default in case of cache miss.
+     * @throws \Psr\SimpleCache\InvalidArgumentException
+     *   MUST be thrown if the $key string is not a legal value.
+     */public function get($key, $default = null)
+    {
+        return null;
+    }
+
+    /**
+     * Persists data in the cache, uniquely referenced by a key with an optional expiration TTL time.
+     *
+     * @param  string $key The key of the item to store.
+     * @param  mixed $value The value of the item to store, must be serializable.
+     * @param  null|int|\DateInterval $ttl Optional. The TTL value of this item. If no value is sent and
+     *                                      the driver supports TTL then the library may set a default value
+     *                                      for it or let the driver take care of that.
+     * @return bool True on success and false on failure.
+     * @throws \Psr\SimpleCache\InvalidArgumentException
+     *   MUST be thrown if the $key string is not a legal value.
+     */
+    public function set($key, $value, $ttl = null)
+    {
+        return false;
```

Comment:
False indicates an error. I think it'd be better to have all functions simulate a happy flow. So make this null-cache pretend everything's working. The only exception here is the `has()` method.